hooks/command

```diff
@@ -200,7 +200,10 @@ else
       echo "Warning: init container failed with exit code $init_container_status, this usually indicates plugin misconfiguration or infrastructure failure"
       status="$init_container_status"
     else
-      status="$(echo "$pod_json" | jq ".status.containerStatuses[0].state.terminated.exitCode")"
+      container_status="$(echo "$pod_json" | jq -e ".status.containerStatuses[0].state.terminated.exitCode")"
+      if $? -eq 0; then
```

Comment:
I guess at this point we could also compare null here

Reply:
```suggestion
      if [[ $? -eq 0 ]]; then
```

Reply:
@iffyio FYI looks like this broken shellcheck on master

Reply:
https://github.com/EmbarkStudios/k8s-buildkite-plugin/pull/42